packages/types/src/request.ts

```diff
@@ -1,6 +1,19 @@
-/**
- * Request data included in an event as sent to Sentry
- */
+/** Possible SentryRequest types that can be used to make a distinction between Sentry features */
+export type SentryRequestType = 'event' | 'transaction' | 'session';
+
+/** A generic client request. */
+export interface SentryRequest {
+  body: string;
+  type: SentryRequestType;
+  url: string;
+  // headers would contain auth & content-type headers for @sentry/node, but
+  // since @sentry/browser avoids custom headers to prevent CORS preflight
+  // requests, we can use the same approach for @sentry/browser and @sentry/node
+  // for simplicity -- no headers involved.
+  // headers: { [key: string]: string };
```

Comment:
Maybe it's time to get rid of this instead of moving it around. I already don't remember why we commented it out, it is unlikely we will bring it back.